certification/internal/policy/new_policy_template.go

```diff
@@ -0,0 +1,35 @@
+package policy
```

Comment:
This file needs to be updated based on this PR once it is merged: https://github.com/komish/preflight/pull/7

Reply:
Also, I feel like this should be in a different directory? Like 'hack/'? I don't think we want it compiled in by default.

Reply:
will do.

one general concept question -

// +preflight:codegen:replace-template-with-actual-name
// +preflight:codegen:todos-prompt

can be something that will be implemented,  or not in scope for now?

Reply:
Let's pull this template and move the discussion on code generation to an issue for now so that we can hack out the logistics.

Reply:
Not really in scope right now.